drop-boxes/register-nanopore-dropbox/register-nanopore.py

```diff
@@ -132,44 +129,75 @@ def createExperimentFromMeasurement(transaction, currentPath, space, project, me
     runExperiment.setPropertyValue("Q_NANOPORE_HOSTNAME", measurement.getMachineHost())
     runExperiment.setPropertyValue("Q_DATA_GENERATION_FACILITY", origin)
     runExperiment.setPropertyValue("Q_MEASUREMENT_START_DATE", convertTime(measurement.getStartDate()))
-    #if measurement.getAdapter():
-    #    runExperiment.setPropertyValue("Q_SEQUENCING_ADAPTER", measurement.getAdapter())
-    # runExperiment.setPropertyValue("Q_EXTERNALDB_ID",) best skip and parse sample information at sample level, no experiment-wide ID from what I can tell
+    if measurement.getAdapter():
+        runExperiment.setPropertyValue("Q_SEQUENCING_ADAPTER", measurement.getAdapter())
     # handle measured samples
     for barcode in rawDataPerSample.keySet():
+        print "handling barcode: "+barcode
         datamap = rawDataPerSample.get(barcode)
         newLogFolder = createLogFolder(currentPath)
         copyLogFilesTo(measurement.getLogFiles(), currentPath, newLogFolder)
         createSampleWithData(transaction, space, barcode, datamap, runExperiment, currentPath, newLogFolder)
 
+# fills the global dictionary containing all checksums for paths from the global checksum file
+# 20886d8be5b6e899c707292f942ccb2edb3dd5dd69df757ed029d9f3200fa95a *20200304185641_QNANO038AT_E19D023c02_PAE34304/20200219_1107_2-A3-D3_PAE34304_6351def9/fastq_fail/PAE34...
+def fillChecksumMap(checksumFilePath):
+    with open(checksumFilePath, 'r') as chf:
+        for line in chf:
+            tokens = line.strip().split(" *")
+            print tokens
+            checksumMap[tokens[1]] = tokens[0]
```

Comment:
maybe comment on how a typical `line` is transformed into a token and what token[1] and token[0] actually are

Reply:
added comments and changed var names